backend/djangoindia/bg_tasks/send_update.py

```diff
@@ -12,20 +15,36 @@ def send_mass_update_email_task(update_id):
         update = Update.objects.get(id=update_id)
```

Comment:
You can also use `update = Update.objects.prefetch("recipients").get(id=update_id)`

for optimisation as `update.recipients.all()` would make another db call.